[debug] [2018-08-15T17:35:54.936Z] ----------------------------------------------------------------------
[debug] [2018-08-15T17:35:54.937Z] Command:       /usr/local/Cellar/node/8.9.2/bin/node /usr/local/bin/firebase deploy
[debug] [2018-08-15T17:35:54.938Z] CLI Version:   4.1.1
[debug] [2018-08-15T17:35:54.938Z] Platform:      darwin
[debug] [2018-08-15T17:35:54.938Z] Node Version:  v8.9.2
[debug] [2018-08-15T17:35:54.939Z] Time:          Wed Aug 15 2018 13:35:54 GMT-0400 (EDT)
[debug] [2018-08-15T17:35:54.939Z] ----------------------------------------------------------------------
[debug] 
[debug] [2018-08-15T17:35:54.947Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2018-08-15T17:35:54.947Z] > authorizing via signed-in user
[debug] [2018-08-15T17:35:54.949Z] > refreshing access token with scopes: ["email","https://www.googleapis.com/auth/cloud-platform","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","openid"]
[debug] [2018-08-15T17:35:54.949Z] >>> HTTP REQUEST POST https://www.googleapis.com/oauth2/v3/token  

[debug] [2018-08-15T17:35:55.093Z] <<< HTTP RESPONSE 200
[debug] [2018-08-15T17:35:55.108Z] >>> HTTP REQUEST GET https://admin.firebase.com/v1/projects/todolistapp-fda9f  
 
[debug] [2018-08-15T17:35:55.410Z] <<< HTTP RESPONSE 200
[debug] [2018-08-15T17:35:55.411Z] >>> HTTP REQUEST GET https://admin.firebase.com/v1/database/todolistapp-fda9f/tokens  
 
[debug] [2018-08-15T17:35:55.759Z] <<< HTTP RESPONSE 200
[info] 
[info] === Deploying to 'todolistapp-fda9f'...
[info] 
[info] i  deploying storage, hosting
[debug] [2018-08-15T17:35:56.028Z] >>> HTTP REQUEST GET https://appengine.googleapis.com/v1/apps/todolistapp-fda9f  
 
[debug] [2018-08-15T17:35:56.429Z] <<< HTTP RESPONSE 500
[debug] [2018-08-15T17:35:56.430Z] <<< HTTP RESPONSE BODY
[info] 

There was an issue deploying your functions. Verify that your project has a Google App Engine instance setup at https://console.cloud.google.com/appengine and try again. If this issue persists, please contact support.
[error] 
[error] Error: HTTP Error: 500, Internal error encountered.
[debug] [2018-08-15T17:35:56.531Z] Error Context: {
  "body": {
    "error": {
      "code": 500,
      "message": "Internal error encountered.",
      "status": "INTERNAL"
    }
  },
  "response": {
    "statusCode": 500,
    "body": {
      "error": {
        "code": 500,
        "message": "Internal error encountered.",
        "status": "INTERNAL"
      }
    },
    "headers": {
      "vary": "X-Origin, Referer, Origin,Accept-Encoding",
      "content-type": "application/json; charset=UTF-8",
      "date": "Wed, 15 Aug 2018 17:35:56 GMT",
      "server": "ESF",
      "cache-control": "private",
      "x-xss-protection": "1; mode=block",
      "x-frame-options": "SAMEORIGIN",
      "x-content-type-options": "nosniff",
      "alt-svc": "quic=\":443\"; ma=2592000; v=\"44,43,39,35\"",
      "accept-ranges": "none",
      "connection": "close"
    },
    "request": {
      "uri": {
        "protocol": "https:",
        "slashes": true,
        "auth": null,
        "host": "appengine.googleapis.com",
        "port": 443,
        "hostname": "appengine.googleapis.com",
        "hash": null,
        "search": null,
        "query": null,
        "pathname": "/v1/apps/todolistapp-fda9f",
        "path": "/v1/apps/todolistapp-fda9f",
        "href": "https://appengine.googleapis.com/v1/apps/todolistapp-fda9f"
      },
      "method": "GET"
    }
  }
}
